tests/ClientTest.php

```diff
@@ -210,6 +207,24 @@ public function join($timeout = null)
     }
 }
 
+class Dummy_Serializer implements \Raven\SerializerInterface
```

Comment:
The `ClientTest::testOurSerializer` method is actually testing the value returned from the serializer instance. I think that this is not appropriate, as the test class about the serializer should do these types of assertions. Instead, here we should check if the serializer is correctly set using the options/setters and if it's called. This is what mocks are for, they verify that something happens. PHPUnit doc is great and has a lot of examples on how to do it